embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/logs/LogSinkImpl.kt

```diff
@@ -0,0 +1,34 @@
+package io.embrace.android.embracesdk.internal.logs
+
+import io.opentelemetry.sdk.common.CompletableResultCode
+import io.opentelemetry.sdk.logs.data.LogRecordData
+
+internal class LogSinkImpl : LogSink {
+    private val storedLogs: MutableList<EmbraceLogRecordData> = mutableListOf()
+
+    override fun storeLogs(logs: List<LogRecordData>): CompletableResultCode {
+        try {
+            synchronized(storedLogs) {
```

Comment:
This works for now. After we ship, both this and `SpansSink` should figure out a common buffering strategy so we don't have to lock when we are writing to the sink.